toolkit/tools/depsearch/depsearch.go

```diff
@@ -66,6 +68,17 @@ func main() {
 		logger.Log.Fatalf("Invalid max depth '%d', valid ranges are -1, >=1", *maxDepth)
 	}
 
+	// Only one of runtimeOnlyPlusBuild or runtimeOnly can be set
+	if *runtimeOnly && *runtimeOnlyPlusBuild {
```

Comment:
Wondering on a higher level ... instead of fatal-ing out ... might as well let it continue.
The user wants `runtime` & `runtime-plus-build` -> The union happens to be `runtime-plus-build`.

Reply:
Considering just making this an int param, let the user select.